issue/models.py

```diff
@@ -278,7 +278,7 @@ def get(
     ):
         if value and language:
             return cls.objects.get(plain_text=value, language=language)
-        raise TypeError("TocSections.get requires value and language paramenters")
+        raise TocSectionGetError()
```

Comment:
Troque:
`raise TocSectionGetError()`

por

`raise TocSectionGetError("TocSections.get requires value and language paramenters")`